hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/OMDBUpdatesHandler.java

```diff
@@ -153,8 +153,18 @@ private void processEvent(int cfIndex, byte[] keyBytes, byte[]
           }
         }
       } else if (action.equals(DELETE)) {
-        if (oldValue != null && !omUpdateEventValidator.isValidEvent(tableName,
-            oldValue, key, action)) {
+        if (null == oldValue) {
+          String keyStr = "";
+          if (key instanceof String) {
+            keyStr = (String) key;
+          }
+          LOG.info(
+              "WARNING !!!, Value of Key: {} in table: {} should not be null " +
+                  "for DELETE event ", keyStr, tableName);
```

Comment:
> I think converting the key to string and the keyStr variable is not needed. You could use key.toString() here instead. (omUpdateEventValidator.isValidEvent() does exectly this a few lines later)

Thanks @Galsza for review, as per my understanding DTOKEN_TABLE is using OzoneTokenIdentifier as key instead of String, so I believe we still need to check type of key, but yes we can simply do toString()... mostly this `isValidEvent` method has not returned false, so log.warn here not hitting else this might throw exception... What do you think ?